easybuild/easyblocks/o/openfoam.py

```diff
@@ -281,6 +281,9 @@ def build_step(self):
                 "Cleaning .*",
             ]
             run_cmd_qa(cmd_tmpl % 'Allwmake.firstInstall', qa, no_qa=noqa, log_all=True, simple=True)
+        elif LooseVersion(self.version) >= LooseVersion('1600'):
```

Comment:
Hmm, this version check looks wrong? OpenFOAM versions are like `2.3.1`, `3.0.1`, etc.

So, the condition here would never be met?